### ISSUE TITLE
app accessibility

### ISSUE THREAD

**Issue by oliashish** [Creatoon/Jar-web-app#64]
@Creatoon I want to work on some improvements for accessibility of the app listed by google lighthouse report.

**Comment by Creatoon** [comment]
@oliashish that will be great, go on.

**Comment by oliashish** [comment]
@Creatoon can I tweak with UI colors and contrast for the same if it is okay?

**Comment by Creatoon** [comment]
@oliashish  Send proper details, if you think UI colors will decrease accessibility.

**Comment by oliashish** [comment]
@Creatoon  low contrast makes if difficult to read text with ease and sometime automatic screen readers fail to recognize the text due to low contrast. 
can also checkout for more details- https://developers.google.com/web/fundamentals/accessibility?utm_source=lighthouse&utm_medium=devtools

**Comment by Creatoon** [comment]
@oliashish it will be great if you handle this thinkg by day mode, as website is already in dark mode, changing directly the websites color will be tough for both of us, you just have to change the color schema via javascript, what do you say ?

**Comment by oliashish** [comment]
@Creatoon we can do this is dark mode to it's about a comfortable contrast I think, changing the blue text to something more contrast with gray and dark gray background would work I guess, with that we can preserve your vision of dark mode too with less overload. I too am learning new things.

**Comment by Creatoon** [comment]
@oliashish ok , do what you feel right.

**Comment by oliashish** [comment]
@Creatoon few changes made and the accessibility report is - 

from -
 
![Screenshot from 2021-07-02 21-39-28](https://user-images.githubusercontent.com/52878118/124307009-d0317400-db84-11eb-83fe-0aac36571360.png)


to -
 
![Screenshot from 2021-07-02 22-20-42](https://user-images.githubusercontent.com/52878118/124307038-d9badc00-db84-11eb-9eab-5c05136354ca.png)



I hope the report remains the same in production too if my PR passes.

**Comment by oliashish** [comment]
I hope this works for you @Creatoon

**Comment by oliashish** [comment]
@Creatoon 
![Screenshot from 2021-07-02 22-41-58](https://user-images.githubusercontent.com/52878118/124308357-cf99dd00-db86-11eb-90fe-e603538faddd.png)


![Screenshot from 2021-07-02 22-41-58](https://user-images.githubusercontent.com/52878118/124308383-d6285480-db86-11eb-9fb8-1a3da7631571.png)

**Comment by oliashish** [comment]
@Creatoon 

![Screenshot from 2021-07-02 22-43-44](https://user-images.githubusercontent.com/52878118/124308518-03750280-db87-11eb-85be-c123d665f15a.png)
![Screenshot from 2021-07-02 22-43-28](https://user-images.githubusercontent.com/52878118/124308537-07a12000-db87-11eb-8d8b-1aefaa803762.png)
